.changeset/hungry-lizards-float.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/vercel': minor
```

Comment:
```suggestion
'@astrojs/vercel': patch
```

This should be a patch since it refactors internal stuff.

Reply:
You're right, I have updated the branch.